style.css

```diff
@@ -201,10 +201,11 @@ main {
 
 .popup-window {
   position: absolute;
-  width: 1108px;
+  width: 1156px;
   height: 1163px;
   left: 142px;
   top: 77px;
+  border-radius: 16px;
   background-color: #fff;
   z-index: 2;
 }
```

Comment:
- [ ] Please kindly work on the `mobile version` styling of your pop-up modal as currently, it is not well aligned on the `mobile screen size of 375px`

Your design 👇 

![hen-mob](https://user-images.githubusercontent.com/78634841/209138320-baa4fc72-8708-4b5c-b413-73b36d210156.png)

Figma design 👇 

![real-pop](https://user-images.githubusercontent.com/78634841/209138747-3fc1eff1-78c8-456b-938c-5acfb9004119.png)